app/assets/javascripts/demographics_fields.js

```diff
@@ -307,7 +307,10 @@ var PersonValidations = (function (window, undefined) {
     hidden_requireds = $('[required]').not(':visible');
     $('[required]').not(':visible').removeProp('required');
     this_obj.closest('div').find('button[type="submit"]').trigger('click');
-    this_obj.closest('div').find('button[type="submit"]').prop('disabled',true);
+    this_obj
+    .closest('div')
+    .find('.dependent-disable')
+    .attr('disabled', 'disabled');
```

Comment:
should this be `.attr('disabled', true)`?

Reply:
Hey @markgoho just saw this. We identified an issue with the disabled,true where if a user clicked the continue button once, and then had to make changes to the broker registration form, and clicked the button again without reloading the page, the page would remain stuck. This fixes that issue and I verified it locally as well as the cucumber.

Reply:
@jacobkagon I'm referring to the syntax of setting a `disabled` attribute on an element, not the strategy of fixing the issue

It looks like this syntax is fine, though -- cf. https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute#example